.github/workflows/pnp-build-autopilot-processor-nested.yml

```diff
@@ -98,7 +98,12 @@ on:
         type: boolean
         required: false
         default: false
-
+      enable-kafka-streams-lint:
+        description: 'If true runs Kafka Streams Lint'
+        type: boolean
+        required: false
+        default: true
```

Comment:
I'm ok with having it enabled by default, unless it requires some changes in the project/repo to not fail